code/game/objects/items/twohanded.dm

```diff
@@ -868,8 +873,11 @@
 	mobhook.RemoveComponent()
 	user.visible_message("[user] lowers [src].","You lower [src].")
 	item_state = "binoculars"
-	user.regenerate_icons()
 	if(user && user.client)
+		if iscarbon(user)
+			var/mob/living/carbon/human/H = user
```

Comment:
```suggestion
			var/mob/living/carbon/H = user
```
feel free to change the var name to C or something more fitting, or just leave it, doesnt matter